gto-support-api/src/main/java/org/ccci/gto/android/common/api/MultipartFormOutputWriter.java

```diff
@@ -0,0 +1,100 @@
+package org.ccci.gto.android.common.api;
+
+import android.support.annotation.NonNull;
+
+import com.google.common.io.Closer;
+
+import org.ccci.gto.android.common.api.AbstractApi.Request.MediaType;
+import org.ccci.gto.android.common.api.AbstractApi.Request.Parameter;
+import org.ccci.gto.android.common.util.IOUtils;
+
+import java.io.Closeable;
+import java.io.DataOutputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+final class MultipartFormOutputWriter implements Closeable {
+    private static final String PREFIX = "--";
+    private static final String NEWLINE = "\r\n";
+
+    @NonNull
+    private final DataOutputStream mOut;
+    @NonNull
+    private final String mBoundary;
+
+    MultipartFormOutputWriter(@NonNull final OutputStream os, @NonNull final String boundary) {
```

Comment:
I guess it's b/c it's only used in a class in this same package.  I tend to create constructors public unless there's some reason not to..  but I think this is totally fine.